apps/etterlatte-oppdater-behandling/src/main/kotlin/inntektsjustering/OppdaterInntektsjusteringBrevDistribuert.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.etterlatte.inntektsjustering
+
+import no.nav.etterlatte.BehandlingService
+import no.nav.etterlatte.brev.BrevHendelseType
+import no.nav.etterlatte.brev.Brevkoder
+import no.nav.etterlatte.libs.common.feilhaandtering.InternfeilException
+import no.nav.etterlatte.libs.common.sak.KjoeringStatus
+import no.nav.etterlatte.rapidsandrivers.BREV_KODE
+import no.nav.etterlatte.rapidsandrivers.Kontekst
+import no.nav.etterlatte.rapidsandrivers.ListenerMedLogging
+import no.nav.etterlatte.rapidsandrivers.SAK_ID_KEY
+import no.nav.etterlatte.rapidsandrivers.brevId
+import no.nav.etterlatte.rapidsandrivers.brevKode
+import no.nav.etterlatte.rapidsandrivers.sakId
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import org.slf4j.LoggerFactory
+import rapidsandrivers.RapidEvents.KJOERING
+import rapidsandrivers.kjoering
+
+class OppdaterInntektsjusteringException(
+    override val detail: String,
+    override val cause: Throwable?,
+) : InternfeilException(detail, cause)
+
+internal class OppdaterInntektsjusteringBrevDistribuert(
+    rapidsConnection: RapidsConnection,
+    private val behandlingService: BehandlingService,
+) : ListenerMedLogging() {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    init {
+        initialiserRiver(rapidsConnection, BrevHendelseType.DISTRIBUERT) {
+            validate { it.requireKey(SAK_ID_KEY) }
+            validate { it.requireKey(BREV_KODE) }
```

Comment:
Eventuelt her kan du bruke `it.requireValue(BREV_KODE, BREVKODER.OMS_INNTEKTSJUSTERING.name)`, så slepp du if-inga under.